libraries/architecture/src/main/kotlin/io/element/android/libraries/architecture/animation/ScreenTransition.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.libraries.architecture.animation
+
+import androidx.compose.animation.core.Spring
+import androidx.compose.animation.core.spring
+import androidx.compose.runtime.Composable
+import com.bumble.appyx.core.navigation.transition.ModifierTransitionHandler
+import com.bumble.appyx.navmodel.backstack.BackStack
+import com.bumble.appyx.navmodel.backstack.transitionhandler.rememberBackstackSlider
+
+@Composable
+fun <NavTarget> getDefaultTransitionHandler(): ModifierTransitionHandler<NavTarget, BackStack.State> {
```

Comment:
I think it should be called `rememberDefaultTransitionHandler` to make it clear

Reply:
Naming convention is a bit mixing up here, since this method also returns something. So it does something (`remember`) and it return something (`get`).
But I think you are right, on Compose, it's better to use `remember` here.

Reply:
Yes all the `remember` methods return something :-)